src/test-fn/kotlin/com/anaplan/engineering/vdmgradleplugin/TestTest.kt

```diff
@@ -64,7 +64,9 @@ class TestTest(
             tasks = arrayOf("test"),
             fail = !expectSuccess)
         val junitFile = File(dir, "build/vdm/junitreports/TEST-TestTest.xml")
-        Assert.assertTrue(junitFile.exists())
+        val junitFileA = File(dir, "a/build/vdm/junitreports/TEST-TestTestA.xml")
+        val junitFileB = File(dir, "b/build/vdm/junitreports/TEST-TestTestB.xml")
+        Assert.assertTrue(junitFile.exists() || (junitFileA.exists() && junitFileB.exists()))
```

Comment:
Needed for `multiWithDependencyPassingTest`. Otherwise we could traverse the file tree, but that seems needlessly expensive.